client/src/components/Pages/Home/index.js

```diff
@@ -153,13 +152,25 @@ const Home = (props) => {
     </React.Fragment>
   );
 
+  const renderEmptyView = () => (
```

Comment:
The structure is very similar to the other empty pages so probably it would be possible to unify them some way. But I decided to keep it simple and flexible so I let it be this way